src/components/posts/buttons/LikeDeleteFunctions.js

```diff
@@ -0,0 +1,40 @@
+function LikeHandler(
```

Comment:
Do you need all of these input types for this function, in general you shouldn't have unused inputs to functions

Reply:
The problem I was having was that it was parsing the arguments as an array, rather than as props, and so removing the unused arguments was throwing errors/making other arguments incorrect, and so it was easier/more manageable to keep the unused arguments in